client/packages/common/src/ui/components/inputs/DatePickers/BaseDatePickerInput/BaseDatePickerInput.tsx

```diff
@@ -2,13 +2,45 @@ import React, { FC } from 'react';
 import DatePicker, { DatePickerProps } from '@mui/lab/DatePicker';
 import { BasicTextInput } from '../../TextInput/BasicTextInput';
 import { StandardTextFieldProps, TextFieldProps } from '@mui/material';
+import { useAppTheme } from '@common/styles';
 
 export const BaseDatePickerInput: FC<
   Omit<DatePickerProps<Date>, 'renderInput'>
 > = props => {
+  const theme = useAppTheme();
+  
   return (
     <DatePicker
       disabled={props.disabled}
+      PopperProps={{
+        sx: {
+          '& .MuiTypography-root.Mui-selected': {
+            backgroundColor: `${theme.palette.secondary.main}`,
```

Comment:
oh! just after I asked to merge!
there's no need to import theme and do it this way - if you are in the `sx` prop then the palette can be accessed directly. This works for me:

```
<DatePicker
      disabled={props.disabled}
      PopperProps={{
        sx: {
          '& .MuiTypography-root.Mui-selected': {
            backgroundColor: 'secondary.main',
          },
          '& .MuiTypography-root.Mui-selected:hover': {
            backgroundColor: 'secondary.main',
          },
          '& .Mui-selected:focus': {
            backgroundColor: 'secondary.main',
          },
          '& .MuiPickersDay-root.Mui-selected': {
            backgroundColor: 'secondary.main',
          },
        },
      }}
      PaperProps={{
        sx: {
          '& .Mui-selected': {
            backgroundColor: 'secondary.main',
          },
          '& button.Mui-selected:focus': {
            backgroundColor: 'secondary.main',
          },
          '& .Mui-selected:hover': {
            backgroundColor: 'secondary.main',
          },
        },
      }}
```

Reply:
I can create a quick PR to merge it like that (didn't realise you could do it that way 🙇🏻